tensorflow/java/src/main/native/graph_jni.cc

```diff
@@ -163,9 +163,17 @@ Java_org_tensorflow_Graph_addGradients(JNIEnv* env, jclass clazz, jlong handle,
   }
   if (env->ExceptionCheck()) return nullptr;
 
+  jboolean is_copy;
```

Comment:
Why is `is_copy` needed? Can provide `nullptr` to `GetStringUTFChars` right?
(https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/functions.html)